### PR TITLE
Pretty print debug information into translated GLSL files.

### PR DESCRIPTION
**Pull request by dsrbecky**

### CODE REVIEW

**Comment by dsrbecky** [comment]
Where? ;-D

File: gapis/shadertools/cc/spv_manager.cpp [hunk 1]
```diff
@@ -819,14 +836,19 @@ void SpvManager::appendDebugInstruction(std::vector<instruction_t>* debugs, Inst
 
       i.name = new char[str_name.length() + 1];
```
Comment (ben-clayton) [inline gapis/shadertools/cc/spv_manager.cpp]: What's freeing this allocation?

Comment (dsrbecky) [inline gapis/shadertools/cc/spv_manager.cpp]: deleteGlslCodeWithDebug in libmanager

Comment (ben-clayton) [inline gapis/shadertools/cc/spv_manager.cpp]: Fair enough.

**Pull request merged**